server/taskflows/hpccloud/taskflow/pyfr.py

```diff
@@ -111,25 +89,7 @@ def start(self, *args, **kwargs):
 
```

Comment:
The assignment "kwargs['image_spec'] = image_spec" appears in two [places](https://github.com/Kitware/HPCCloud/blob/cluster-provision-refactor/server/taskflows/hpccloud/taskflow/paraview.py#L63-L65) in paraview.py. In the first location it is contained in the "if '_id' not in kwargs['cluster']:" if block. Based on your response above I assume to should not be in that if block.

There is only one "kwargs['image_spec'] = image_spec" [assignment](https://github.com/Kitware/HPCCloud/blob/cluster-provision-refactor/server/taskflows/hpccloud/taskflow/pyfr.py#L88) in pyfr.py but it is also inside a similar if block. Should it be in that if block or should that assignment always happen?